ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -682,28 +729,65 @@ const NetworksForm = ({
           onChange={(value) => {
             setIsEditing(true);
             setChainId(value);
+            autoSuggestTicker(value);
           }}
           titleText={t('chainId')}
           value={chainId}
           disabled={viewOnly}
           tooltipText={viewOnly ? null : t('networkSettingsChainIdDescription')}
         />
-        <FormField
-          warning={warnings.ticker?.msg || ''}
-          warningProps={{
-            'data-testid': 'network-form-ticker-warning',
-            style: {
-              color: 'var(--color-warning-default)',
-            },
-          }}
-          onChange={(value) => {
+        <FormTextField
+          data-testid="network-form-ticker"
+          helpText={
+            suggestedTicker ? (
+              <Text
+                as="span"
+                variant={TextVariant.bodySm}
+                color={TextColor.textDefault}
+              >
+                {t('suggestedTokenSymbol')}{' '}
+                <ButtonLink
+                  as="button"
+                  variant={TextVariant.bodySm}
+                  color={TextColor.primaryDefault}
+                  onClick={() => {
+                    setTicker(suggestedTicker);
+                  }}
+                  style={{ verticalAlign: 'baseline' }}
+                >
+                  {suggestedTicker}
+                </ButtonLink>
+              </Text>
+            ) : null
+          }
+          onChange={(e) => {
             setIsEditing(true);
-            setTicker(value);
+            setTicker(e.target.value);
+          }}
+          label={t('currencySymbol')}
+          labelProps={{
+            variant: TextVariant.bodySm,
+            fontWeight: FontWeight.Bold,
+            paddingBottom: 1,
+            paddingTop: 1,
+          }}
+          inputProps={{
+            paddingLeft: 2,
+            variant: TextVariant.bodySm,
+            'data-testid': 'network-form-ticker-input',
           }}
-          titleText={t('currencySymbol')}
           value={ticker}
           disabled={viewOnly}
         />
+        {warnings.ticker?.msg ? (
+          <HelpText
+            severity={HelpTextSeverity.Warning}
+            marginTop={1}
+            data-testid="network-form-ticker-warning"
+          >
+            {warnings.ticker?.msg}
```

Comment:
At this point, we shouldn't check it
```suggestion
            {warnings.ticker.msg}
```